test/Serilog.Settings.Configuration.Tests/ConfigurationSettingsTests.cs

```diff
@@ -194,6 +194,39 @@ public void TestMinimumLevelOverrides()
             Assert.NotNull(evt);
         }
 
+        [Fact]
+        public void TestMinimumLevelOverridesForChildContext()
+        {
+            var json = @"{
+                ""Serilog"": {
+                    ""MinimumLevel"" : {
+                        ""Default"" : ""Warning"",
+                        ""Override"" : {
+                            ""System"" : ""Warning"",
+                            ""System.Threading"": ""Debug""
+                        }
+                    }        
+                }
+            }";
+
+            LogEvent evt = null;
+
+            var log = ConfigFromJson(json)
+                .WriteTo.Sink(new DelegatingSink(e => evt = e))
+                .CreateLogger();
+
+            log.Write(Some.DebugEvent());
+            Assert.Null(evt);
+
+            var custom = log.ForContext("SourceContext", "System.Threading.Tasks.Task<42>");
```

Comment:
is there any reason to not use 

```cs
log.ForContext<SystemThreading.Tasks.Task<Int32>>();
```
(or something similar)

Reply:
We use `MyGame.Room<489414134>`.  So not type, but value.

Reply:
There is seems issue in `serilog-sinks-file` as it does not work with child matching.

Reply:
oh ok, fair enough :P 
You could be using `Serilog.Core.Constants.SourceContextPropertyName` instead of `"SourceContext"` then ;)